integration/dataproc/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+ALLUXIO_DOWNLOAD_URL=${1}
+
+if [[ -z ${ALLUXIO_DOWNLOAD_URL} ]]; then
+  echo "Alluxio tarball URL cannot be empty"
+  exit 1
+fi
+
+# replace ALLUXIO_DOWNLOAD_URL in dataproc files
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  sed -i '' -e "s|^readonly ALLUXIO_DOWNLOAD_URL.*\$|readonly ALLUXIO_DOWNLOAD_URL=\"${ALLUXIO_DOWNLOAD_URL}\"|" ${DIR}/*
+else
+  sed -i -e "s|^readonly ALLUXIO_DOWNLOAD_URL.*\$|readonly ALLUXIO_DOWNLOAD_URL=\"${ALLUXIO_DOWNLOAD_URL}\"|" ${DIR}/*
```

Comment:
if the file is updated in the future in a way that results in this sed operation doing zero replacements, is there a way to detect and trigger failure? trying to avoid the scenario where things get refactored and silently fail

Reply:
ping on this comment

Reply:
i'll diff the `.generated` files with the original to see if changes were made

Reply:
maybe a grep for the matching string before the sed is easier? you could replace the url with the same url that was there and result in no diff

Reply:
ok sounds good